main/wallet-backend.js

```diff
@@ -69,119 +83,160 @@ class WalletBackend {
       }
     }
 
-    seed = generateMnemonic()
+    seed = ethers.Wallet.createRandom().mnemonic.phrase
     if (!this.disableKeytar) {
       await keytar.setPassword(service, 'seed', seed)
     }
     return { seed, isNew: true }
   }
 
-  async getAddress () {
-    assert(this.provider)
-    const [address] = await this.provider.wallet.getAccounts(
-      0,
-      1,
-      CoinType.MAIN
-    )
-    return address
+  async fetchBalance () {
+    assert(this.signer)
+    return await this.signer.getBalance()
   }
 
-  async fetchBalance () {
-    assert(this.provider)
-    assert(this.address)
-    return await this.provider.getBalance(this.address)
+  /**
+   * @param {string} to
+   * @param {ethers.BigNumber} amount
+   * @returns {Promise<string>}
+   */
+  async transferFunds (to, amount) {
+    console.log('transferFunds()', { to, amount: amount.toString() })
```

Comment:
Do we want to make this more human readable if we are using the console to show transfers?

Reply:
Actually, I think we shouldn't be using `console.log`, I think that does not end up being saved in log files and therefore these messages are lost when running the app in the tray. We should use `electron-log` instead.

Reply:
Good idea, integrated `electron-log`. @patrickwoodhead only devs see these, so JSON formatting is ok